gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -204,10 +203,6 @@ void GazeboRosApiPlugin::loadGazeboRosApiPlugin(std::string world_name)
   pub_link_states_connection_count_ = 0;
   pub_model_states_connection_count_ = 0;
 
-  /// \brief advertise all services
-  if (enable_ros_network_)
-    advertiseServices();
-
```

Comment:
By moving this from here to after we advertise the clock, we get mostly the same behavior as was duplicated in the `advertiseServices` method.  One thing I mentioned to @scpeters in person is that this slightly changes the order of operations, since before we would advertise the clock and then get the parameters later, whereas now we are doing it all at the same time.  I don't have a strong reason why that would matter, but I thought I would point it out.